error_handler.py

```
@@ -0,0 +1,273 @@
+"""
+
+import logging
+import sqlite3
+import discord
+import asyncio
+from typing import Optional, Union, Type, Callable, Any
+from functools import wraps
+
+logger = logging.getLogger('discord_bot.error_handler')
+
+class ErrorSeverity:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)